hooks/hookstage/allprocbidresponses.go

```diff
@@ -0,0 +1,17 @@
+package hookstage
+
+import (
+	"context"
+)
+
+type AllProcessedBidResponses interface {
```

Comment:
It would be good if we could add go doc comments for the public interfaces and types in the hookstage package files too.

For the payload type it might be good to mention this is what is modified by the hook interface implementation for the given stage.